### PR TITLE
DDF-2443: Ignores one unit test depending on specific log output and comments others

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
These tests are very fragile - as evidenced by the fact one test is broken after logging changes have been made - and should not be expecting specific log output. In order to fix the broken build, this change ignores the broken test. The associated ticket will remain open and the other tests will be fixed in short order.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@brendan-hofmann @rzwiefel 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@beyelerb
@stustison

#### How should this be tested?
Full build/test run to ensure the unit test is not causing the build to fail.

#### Any background context you want to provide?
Logging recently went through an overhaul/cleanup and fragile tests like these are prone to break under those changes.

#### What are the relevant tickets?
DDF-2443

#### Screenshots (if appropriate)
N/A

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java [hunk 1]
```diff
@@ -234,6 +234,7 @@ public void testApplicationUploadEndpointCreateApplicationServiceException() thr
      *
      * @throws Exception
      */
+    // TODO RAP 29 Aug 16: DDF-2443 - Fix test to not depend on specific log output
```
Comment (brendan-hofmann) [inline platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java]: What is 'RAP 29'?

Comment (coyotesqrl) [inline platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java]: Initials and date `RAP 29 Aug 16`

**Comment by brendan-hofmann** [comment]
Lgtm other than that one question. 👍

File: platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java (hunk 1 above)
Comment (brendan-hofmann) [inline platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java]: Of course. Clearly it's time to go home. 😄

**Comment by peterhuffer** [comment]
LGTM 👍

**Comment by adimka** [comment]
Hero'd 👍

**Comment by coyotesqrl** [comment]
@beyelerb & @stustison Since this is breaking master, could I get a quick review/approval please?

**Comment by coyotesqrl** [comment]
Bamboo completed build successfully then had an OutOfMemoryError.

**Pull request merged**